cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/AnalyzerConfig.java

```diff
@@ -334,6 +338,22 @@ private AnalyzerConfig() {
   public static final String ALLOW_CAPACITY_ESTIMATION_ON_PROPOSAL_PRECOMPUTE_DOC = "The flag to indicate whether to "
       + "allow capacity estimation on proposal precomputation.";
 
+  /**
+   * <code>topics.with.min.leaders.per.broker</code>
+   */
+  public static final String TOPICS_WITH_MIN_LEADERS_PER_BROKER_CONFIG = "topics.with.min.leaders.per.broker";
+  public static final String DEFAULT_TOPICS_WITH_MIN_LEADERS_PER_BROKER = "";
+  public static final String DEFAULT_TOPICS_WITH_MIN_LEADERS_PER_BROKER_DOC = "The topics that should have a minimum "
+      + "number of replica on brokers that are not excluded for replica move. It is a regex.";
+
+  /**
+   * <code>min.topic.leaders.per.broker</code>
+   */
+  public static final String MIN_TOPIC_LEADERS_PER_BROKER_CONFIG = "min.topic.leaders.per.broker";
+  public static final int DEFAULT_MIN_TOPIC_LEADERS_PER_BROKER = 1;
+  public static final String MIN_TOPIC_LEADERS_PER_BROKER_DOC = "The minimum required number of leader per broker"
+      + " for topics that must have leader on brokers that are not excluded for replica move.";
+
```

Comment:
Can we make sure that `Configurations.md` has the same documentation as the ones listed here for these two new configs.

Currently they are inconsistent.